src/Executor.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // Executor This struct ensures the parallel execution of all command executions
 type Executor struct {
-	planets []Planet
+	planets []*Planet
```

Comment:
even if the closure avoiding copy is created like this
planet := &planet
the changes still wouldn't be permanent